posthog/tasks/exports/image_exporter.py

```diff
@@ -191,6 +191,7 @@ def export_image(exported_asset: ExportedAsset) -> None:
                     process_query_dict(
                         exported_asset.team,
                         exported_asset.insight.query,
+                        dashboard_filters_json=exported_asset.dashboard.filters if exported_asset.dashboard else None,
```

Comment:
non-blocking fly-by that this should have been caught by tests 😊

Reply:
Do you mean specific ones or "should" as in "it would make sense to test"?

I checked the frontend snapshots, but alas they use mocked data of course.

Reply:
I mean we have core functionality of the platform that we were able to break, that clearly signals a missing test at some level.

IMO The backend tests should have protected us here.